artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java

```diff
@@ -363,49 +355,43 @@ public Executor getExecutor() {
    }
 
    @Override
-   public void stop() throws Exception {
-      if (stopping) {
-         return;
+   public synchronized void stop() throws Exception {
+      if (state == State.STOPPING || state == State.STOPPED || state == State.PAUSING) {
+         logger.debug("Bridge {} state is {}. Ignoring call to stop.", configuration.getName(), state);
+         if (state == State.PAUSING) {
+            throw ActiveMQMessageBundle.BUNDLE.bridgeOperationCannotBeExecuted(configuration.getName(), "stopped", state);
+         } else {
+            return;
+         }
```

Comment:
Similar comment to start() around state check

Reply:
Done.